src/models/api/README.md

```
@@ -0,0 +1,5 @@
+# List of source files stored in `src/models/api` directory
+
+## [__init__.py](__init__.py)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)